README.md

```diff
@@ -66,13 +88,105 @@ Further details on creating the study population can be found in the [`OpenSAFEL
 
 The contents of this repository MUST NOT be considered an accurate or valid representation of the study or its purpose. This repository may reflect an incomplete or incorrect analysis with no further ongoing work. The content has ONLY been made public to support the OpenSAFELY [open science and transparency principles](https://www.opensafely.org/about/#contributing-to-best-practice-around-open-science) and to support the sharing of re-usable code for other subsequent users. No clinical, policy or safety conclusions must be drawn from the contents of this repository.
 
-# About the OpenSAFELY framework
-
-The OpenSAFELY framework is a Trusted Research Environment (TRE) for electronic health records research in the NHS, with a focus on public accountability and research quality.
-
-Read more at [OpenSAFELY.org](https://opensafely.org).
-
-# Licences
+## Output
+
+### consort_\*.csv
+
+| Variable           | Description                                                    |
+|--------------------|----------------------------------------------------------------|
+|     Description    | criterion applied to cohort                                    |
+|     N_midpoint6    | number of people in the cohort after criterion applied time    |
```

Comment:
At the top of this section add a sentence explaining what _midpoint6 mean (i.e. disclosure control to Ns)

Reply:
Added explanantion and links with information.